diff --git a/src/UpdateCheck.lua b/src/UpdateCheck.lua
index cb74c077..8e2a9e0f 100644
--- a/src/UpdateCheck.lua
+++ b/src/UpdateCheck.lua
@@ -4,6 +4,9 @@
 -- Module: Update Check
 -- Checks for updates
 --
+return "none"
+
+--[[
 local connectionProtocol, proxyURL, noSSL = ...
 
 local xml = require("xml")
@@ -335,3 +338,4 @@ opFile:close()
 
 ConPrintf("Update is ready.")
 return updateMode
+--]]
